addons/website_sale/models/product_attribute.py

```diff
@@ -30,3 +30,18 @@ def _prepare_single_value_for_display(self):
         for ptal in single_value_lines:
             single_value_attributes[ptal.attribute_id] |= ptal
         return single_value_attributes
+
+class ProductAttributeValue(models.Model):
+    _inherit = 'product.attribute.value'
+
+    def is_product_variant_in_category(self, category):
+        # All products category
+        if(not category.name):
```

Comment:
You don't need parentheses in an if :)
```suggestion
        if not category.name:
```

Reply:
```suggestion
        if not category:
```